### ISSUE TITLE
Screenshot is not shown in HTML report if the name contains "#"

### ISSUE THREAD

**Issue by gremjua** [abhishekkyd/protractor-html-reporter-2#7]
I am currently using jasmine-tagged to add #p0 to the it()s I want to run as my sanity test suite.
Since the screenshot generated contains the name of the spec, it will contain "#".
The HTML report will not show the screenshot because of this.

Could we change this behavior? Or provide a way to configure the name of the screenshots the HTML reporter should look for?

Thanks.

**Comment by jamesomorodion** [comment]
@gremjua Is this still a blocker/issue for you?

**Comment by praveendvd** [comment]
@jamesomorodion  yes , and also  if suitename+specname is more than 125 character the test fails
